app/presenters/content_item/withdrawable.rb

```diff
@@ -5,7 +5,7 @@ def withdrawn?
     end
 
     def page_title
-      withdrawn? ? "[Withdrawn] #{title}" : title
+      withdrawn? ? "[Withdrawn] #{withdrawal_context_title}: #{title}" : title
```

Comment:
Good spot in recognising we need to fix the withdrawn content. Unfortunately we have a couple of issues around this change - 
1) It's caused a failing test on fatality notices, usually you'd want to update that test BUT see 2.
2) This is being applied to all content types and government-frontend serves content that is a different format from "publications" and we're scoping this title change to just publications and consultations. Publication subtypes are all the ones updated in the en.yml to have an "overview" entry, or you can see them in whitehall publisher. There is probably something we can query in the content item to determine whether it's a publication - we should make that check here and only update the title if it's one we want to affect. I think this is because only publications and consultations can have attachments, but Tobi will have more background otherwise.

Reply:
> This is being applied to all content types and government-frontend serves content that is a different format from "publications" and we're scoping this title change to just publications and consultations. Publication subtypes are all the ones updated in the en.yml to have an "overview" entry, or you can see them in whitehall publisher. 

So, I was trying to work out what this was scoped to define "publications and consultations" and you've mentioned this defined by the existence of the `overview` entry.  So the logic has been based of this.


> There is probably something we can query in the content item to determine whether it's a publication - we should make that check here and only update the title if it's one we want to affect. I think this is because only publications and consultations can have attachments, but Tobi will have more background otherwise.

I've used the `overview` property in this instance. Hopefully that's correct.

Reply:
That should work but it feels a bit dirty/manual it feels ok to check for a translation and display it based on whether it exists, but less so to base other template decisions on it. The content item seems to have an entry of
```schema_name: "publication",```
regardless of the sub type, if you can hook into that it would be cleaner.

Reply:
🙌🏼 Ace, that works - updated.